src/Frontend/Page/Idea.hs

```diff
@@ -104,6 +107,13 @@ data ReportComment = ReportComment Comment
 
 instance Page ReportComment where
 
+data EditComment
+    = EditComment Idea Comment
```

Comment:
Let's ignore these questions for now.  Should happen naturally in a bigger comment refactoring.